AnalyticTune/AnalyticTune.js

```diff
@@ -990,6 +990,19 @@ function load_log(log_file) {
         flight_data.data[0].y = log.get("SIDD", "Targ")
 
         update_time(SIDD_time)
+
+        const MSG = log.get("MSG")
+        const len = MSG.Message.length
+        for (let i = 0; i < len; i++) {
+            const msg = MSG.Message[i]
+
+            // narrow the SID time window
+            if (msg === 'SystemID Finished') {
```

Comment:
Could we not looks at the last time stamp of the SIDD data? Looking for messages is quite fragile.

Reply:
Unfortunately not because once started the data is being logged until you land. Obviously we could change sysid (and I think we should) but that doesn't help right now

Reply:
Well if you use the parameter for the length of the chirp in the webtool to determine the end of the data then that should work.